### ISSUE TITLE
Admin can assign candidate screenings to screeners

### ISSUE THREAD

**Issue by miranmirza** [miranmirza/screening-webapp#33]
Self explanatory.

**Comment by miranmirza** [comment]
So for this issue, we might need to tweak out models and assign screenings to a candidate (1 candidate has many screenings), then we can just tweak the views to get each screeners screenings and give admins the ability to assign screenings (in the future we'll integrate this with youcanbookme).

**Comment by EmilyMB** [comment]
Yeah, it would be nice if we could figure out an alternative to the free version of youcanbookme that would allow all the screeners' availability to presented on one calendar to the candidate.
